### DISCUSSION TITLE
Can I change the terminal colors?

### DISCUSSION THREAD

**Discussion by stewday** [cockpit-project/cockpit#21447] [Q&A] [answered]
As per the title, is it possible to change the color/opacity of the terminal highlighting, or, failing that, turning it off completely?

Running 287.1-0+deb12u3 (Debian 12)

**Comment by martinpitt** [comment]
You can change the color schema in the terminal by choosing between four themes. But it's not possible to customize it further.

![image](https://github.com/user-attachments/assets/aaa4af42-d066-4d24-8967-d3ebab377c94)

I'm not sure if that selector was already present in version 287. If not, update to the [official backport](https://cockpit-project.org/running.html#debian)

**Reply by stewday** [reply]
Hi Martin, the problem looks like this ...
[image](https://global.discourse-cdn.com/spiceworks/original/4X/2/8/2/282cafd6d685ca7dde1a2425c06a46e6c56cc14a.png)

(from a similar issue posted on [Spiceworks community](https://community.spiceworks.com/t/turning-off-cockpit-terminal-highlighting/686371))

In your previous comment (now removed) you say anything inside the terminal is under the control of _your_ shell, do you mean my client machine or the server? 

My guess is the "highlighting" is set by the LS_COLORS directory _bold_ status. If Cockpit is using the setting from the client or server I can try changing those. Or is the Cockpit terminal its own thing and can't be changed?

**Reply by martinpitt** [reply]
It's under control of the shell of whatever machine you log into. I.e. I suspect what you call the "server". bash looks at `$TERM` and termios etc., unfortunately this has a long history and a lot of baggage..

**Reply by stewday** [reply, marked as answer]
Thanks Martin ... I added a customized list of LS_COLORS to ~/.bashrc and that has resolved the issue (removed BOLD from all combinations).
